bin/uvn.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Script to update version numbers (uvn) in the code. It will create a new
```

Comment:
I think we should update file name from `uvn.sh` to `update_plugin_version_number.sh` which is more descriptive.

Reply:
Merging this as-is, but noting that the name will probably be revised.